hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientReplicatedMapProxy.java

```diff
@@ -102,7 +102,7 @@ public V put(K key, V value, long ttl, TimeUnit timeUnit) {
         Data valueData = toData(value);
         Data keyData = toData(key);
         ClientMessage request = ReplicatedMapPutCodec.encodeRequest(name, keyData, valueData, timeUnit.toMillis(ttl));
-        ClientMessage response = invoke(request);
+        ClientMessage response = invokeOnPartitionOfProxyName(request);
```

Comment:
As far as I understand we can not send to random node anymore,  because that random node might be lite member and does not have any data in it. So what will happen in case of non-smart client. Are users can not access to replicated map anymore if they are using non-smart client ? Or will they be routed to a non-lite member by lite-member ?